README.md

```diff
@@ -1,24 +1,25 @@
 scrapi
 ======
 
-```master``` build status: [![Build Status](https://travis-ci.org/CenterForOpenScience/scrapi.svg?branch=master)](https://travis-ci.org/CenterForOpenScience/scrapi)
+```master``` build status: [![Build Status](https://travis-ci.org/fabianvf/scrapi.svg?branch=master)](https://travis-ci.org/fabianvf/scrapi)
```

Comment:
Not sure what happened here, must have been a bad merge. These should definitely point at CenterForOpenScience/scrapi